### ISSUE TITLE
Validation loss unstable and not decreasing 

### ISSUE THREAD

**Issue by ikramabdel** [keras-team/keras#12614]
Can someone please with the help of the figures below tell me what the problem is with the training 
Input = (12000,144,160,4) and 9000 for training 3000 for validation.

To summarize my model is 2d Unet , loss function is Dice 

![Figure_1](https://user-images.githubusercontent.com/45387325/55464579-b7c7fd00-55fb-11e9-9460-1f10ffd955b5.png)
![Figure_2](https://user-images.githubusercontent.com/45387325/55464578-b7c7fd00-55fb-11e9-8a9f-6b76d3248825.png)

**Comment by tranvohuy** [comment]
You should ask this question on stackoverflow

**Comment by ikramabdel** [comment]
> You should ask this question on stackoverflow

I did aswell . thanks.
